docassemble/AssemblyLine/al_general.py

```diff
@@ -1031,6 +1038,160 @@ def address_block(
             )
         )
 
+    def pronoun(self, **kwargs):
+        """Returns a pronoun as appropriate, based on attributes.
```

Comment:
```suggestion
        """Returns an objective pronoun as appropriate, based on attributes.
```

I'm okay with objective being the default, but it should be clearer that it is.